src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/InputJavadocTagContinuationIndentationDescription.java

```diff
@@ -0,0 +1,30 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoctagcontinuationindentation;
+
+/** Config: default */
+public class InputJavadocTagContinuationIndentationDescription {
+    /**
+     * General Description here.
+     *
+     * @param s
+     * Description 1. // violation
+     * Description 2. // violation
+     * Description 3. // violation
+     *                         Description 4 with extra indentation.
+     *
```

Comment:
Please a test cases:

- with missing asterisks
```java
  * @param s
    Description 1.
```
- with missing space after asterisk(s)
```java
  * @param s
   *Description 1.
   ******Description 2.
```
- with multiple asterisks
```java
  * @param s
  **** Description 1.
```
It is OK to ignore some weird cases (there should be other checks to validate asterisks), but there should be no false positives and no crashes.

Reply:
Okay, I'll add these cases

Reply:
Done 👍 

> with missing asterisks

Only this case is not generating any violations and there are no false positives or crashes